src/transformers/models/llama/modeling_llama.py

```diff
@@ -1180,7 +1183,7 @@ def forward(
 
         loss = None
         if labels is not None:
-            loss = self.loss_function(logits=logits, labels=labels, vocab_size=self.config.vocab_size, **kwargs)
+            loss = self.loss_function(logits=logits, labels=labels, vocab_size=self.config.vocab_size, num_items_in_batch=num_items_in_batch, **kwargs)
```

Comment:
no, the whole point of having kwargs here is not not introduce unwanted training specific args 😓

Reply:
right, the problem is I have to pop it from kwargs to keep from passing it to `model(...)` above